src/views/docs/api/mocks.ejs

```diff
@@ -20,7 +20,8 @@ verification. Mocking requires mountebank to remember information about each req
 For long-running instances of mountebank, it is recommended that you run <code>mb</code> with the
 <code>--datadir</code> command line option so that all requests are persisted to disk. Otherwise,
 mountebank will save them to memory, which will create a significant performance constraint
-when there are a large number of requests saved.</p>
+when there are a large number of requests saved. You can also manually clear an imposter's saved 
+requests - see the <a href='/docs/api/overview'>overview page</a> for details.</p>
```

Comment:
I originally had this linking directly to `/docs/api/overview#delete-requests` but the anchor tag wasn't working properly - it would take you to the page but not even close to the right anchor tag. Couldn't figure out.